### ISSUE TITLE
Page summary: Subtitle marked up as header (1.3.1, 2.4.6)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#856]
### Expected behavior

Heading elements used to mark page/section headings of the page content. 
### Actual behavior

The subtitle element is marked up as an H2 heading element and is not a heading in the page content/organization. It's a label/sub-title for the page. This places the subtitle as a heading in the page hierarchy. See https://www.w3.org/TR/html5/common-idioms.html#common-idioms for some ways to mark up subheadings

**Comment by tealiie** [comment]
Hi, just forked this. I will have a look at this.

**Comment by Blackbaud-PatrickOFriel** [comment]
Great, let us know if you need any help!

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.12.0](https://github.com/blackbaud/skyux/releases/tag/v1.12.0)
